### ISSUE TITLE
'Hidden Items' list is empty when loading up a project

### ISSUE THREAD

**Issue by manspaniel** [Devine-Davies/vs-make-hidden#22]
Hi there! Love this extension — we use it internally to hide seldom-used files in our projects, committing the `.vscode` directory to Git which works super well.

I've noticed that in all projects — new and old — the files are still hidden, but the files are no longer visible under the 'Hidden items' section. It's only been happening for a couple of days.

There is a workaround at the moment, but it's only temporary — when the list is empty, right-clicking a non-hidden file and clicking "Hide" will actually bring the full list back again, but reloading the window results in a blank list again.

Using:
- Make Hidden: v4.0.2
- VSCode: v1.64.2

**Comment by Devine-Davies** [comment]
Hello, thank you for contacting and using the plugin sorry to hear it's not working as expected. Happy to try and get this resolved ASAP.

I've not experienced the type of bug you're referring to and it sounds like a bit of a strange one. I recently updated the plugin so guessing the bug has been introduced here. I'll be sure to debug my end and In the meantime are you able to provide a few more details: 

What OS are you running?
Are you running inside a container or WSL mode? 
Are you using multiple workspaces? 
Are you able to provide a short video/gif of the issue?

Thank you for your time manspaniel and raising the issue.

**Comment by manspaniel** [comment]
Hi Rhys,

Thanks for your reply!

Running directly on MacOS Monterey, in the repository root, which is where the `.vscode` file sits. The `.vscode` folder is committed to Git.

Here's a vid!

https://user-images.githubusercontent.com/1719776/155285829-04b8d1be-a6eb-41e3-b0a0-ed1dc1fa569d.mp4

**Comment by Devine-Davies** [comment]
Hi manspaniel, Cheers for that 👍 

After digging around, i noticed i had removed '*' from activationEvents in the package.json. The reason fro removing was to help performance but it stopped the plugin from running when VSCode had finished loading. I have now updated the package.json to include the correct CMD and this should be working again.

I've published version 4.0.3 of the plugin with the fix, if you update to that and let me know if that fixes the issue that would be great.

Cheers again , much appreciated 👍
